README.md

```diff
@@ -10,153 +14,129 @@
-
-> To get a local copy up and running, follow these steps.
-
-- `git clone` --> Using your terminal
-- `npm install` --> For uploading all the necessary packages.
-
-- `npm start` --> to launch in local server
-
-
-### Prerequisites
-
-> In order to run this project you need:
-
-- GitHub flow
-- Gitflow
-- HTML and CSS
-- JAVASCRIPT (ES6/...)
-- npm
-- node.js
-- Any code editer (vs code)
-
-
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/HTML">REACT</a></li>
-    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/CSS">CSS</a></li>
-    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript">JavaScript (ES6/ES7..)</a></li>
-    <li><a href="https://webpack.js.org/">JSX</a></li>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://webpack.js.org/">webpack</a></li>
   </ul>
 </details>
 
+<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-- **Read with chapster**
-- **Track your reading progress**
+- **Take user input**
+- **Adds books using API**
+- **Removes books using API**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
 ### Prerequisites
 
-> In order to run this project you need:
+In order to run this project you need:
 
-- GitHub flow
-- Gitflow
-- HTML and CSS
-- JAVASCRIPT (ES6/...)
 - npm
-- node.js
+- Node.js
 - Any code editer (vs code)
 
-> Clone this repository to your desired folder:
+### Setup
 
-- `https://github.com/SekoViper/book-store.git`
+Clone this repository to your desired folder:
+
+- Run `git clone https://github.com/SekoViper/book-store.git`
 
 ### Install
 
-> Install this project with:
+Install this project with:
 
-- `npm install`
+```sh
+  cd book-store
+  npm install
+```
 
 ### Usage
 
-> To run the project, execute the following command:
+To run the project, execute the following command:
 
-- Run `npm start` to run app in local server with webpack
+```sh
+  npm start
+```
 
+### Run tests
 
-### Deployment
+```sh
+  npm test
+```
 
-> You can deploy this project using:
+### Deployment
 
-> npm run deploy - Deploys using gh-pages
+`npm run deploy` - Deploys using gh-pages
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHOR -->
-
-👤 **Seko Dameh**
+## 👤 Author
 
 - GitHub: [@SekoViper](https://github.com/SekoViper)
 - Twitter: [@SekoViper](https://twitter.com/SekoViper)
 - LinkedIn: [@livingstone-dameh](https://www.linkedin.com/in/livingstone-dameh-b755a5151/)
 
```

Comment:
- [ ] [optional] @SekoViper great work on your application, it would have been nice if you added the `Future Features` section in your readme file.